docker/run.sh

```diff
@@ -136,17 +168,30 @@ PGPASSWORD=$DB_PASSWORD psql "sslmode=verify-ca sslrootcert=/root/.postgresql/ro
 -c "\copy (SELECT row_to_json(t) FROM (SELECT * FROM casev2.event where event_type!='CASE_CREATED' and event_type!='UAC_UPDATED' and event_type!='SAMPLE_LOADED' and event_type!='RM_UAC_CREATED' and rm_event_processed >= '$START_OF_PERIOD' and rm_event_processed < '$END_OF_PERIOD') t) To '$EVENTS_FILE';"
 
 
-echo "zipping event file"
+if [ -n "$DATAEXPORT_MI_BUCKET_NAME" ]
+then
+      echo "$DATAEXPORT_MI_BUCKET_NAME is empty"
```

Comment:
`-z` means "if env var is empty". `-n` means "if the env var is NOT empty". Your logic says "If the env var is not empty act like it _is_ empty".